src/Home.js

```diff
@@ -23,18 +23,33 @@ function Home() {
             <h1>Res I.P.</h1>
             <h2>pronounced /ˈresəˌpē/</h2>
             <form>
+                {/* creates a text box and submit button on the same line */}
                 <input onChange={(event) => setQuery(event.target.value)} type="text" name="name" placeholder="i.e. Noodles" />
                 <button onClick={handleSubmit}>search</button>
             </form>
-            {recipes.map(element => <Recipe recipe={element} />)}
+            {/* maps over array and applys the Recipe component to each object in the array of objects */}
+            {recipes.map((recipeObject, key) => <Recipe key={key} recipe={recipeObject} />)}
         </div>
     );
 }
 
 function Recipe(props) {
+    const handleSave = (event) => {
+        event.preventDefault();
+        const url = `http://localhost:3001/save`; // database URL
```

Comment:
This is technically not the database URL, this is actually the URL to our server. The server will then communicate with the database

Reply:
Thanks for the clarification.